apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -93,6 +93,13 @@ class BoardAutodetectFailed(Exception):
         default=None,
         help="Port to use for connecting to hardware.",
     ),
+    server.ProjectOption(
+        "serial_number",
+        optional=["open_transport", "flash"],
+        type="str",
+        default=None,
+        help=("Board serial number."),
+    ),
```

Comment:
Can we clarify in the `help` strings how the `port` and `serial_number` options interact? What happens if we pass both options or neither?

Reply:
good point, I added more info to both help messages. It should be clear now